docker/packager/binary/Dockerfile

```diff
@@ -91,6 +91,9 @@ ENV PATH="$PATH:/usr/local/go/bin"
 ENV GOPATH=/workdir/go
 ENV GOCACHE=/workdir/
 
+RUN curl https://raw.githubusercontent.com/matus-chochlik/ctcache/7fd516e91c17779cbc6fc18bd119313d9532dd90/clang-tidy-cache -Lo /usr/bin/clang-tidy-cache \
```

Comment:
I found no official releases for clang-tidy-cache: https://github.com/matus-chochlik/ctcache/releases

Just wondering how you found this binary?

Reply:
The current master, see https://github.com/matus-chochlik/ctcache/blob/7fd516e91c17779cbc6fc18bd119313d9532dd90/clang-tidy-cache

Reply:
Ah, didn't realize it's implemented in Python.